### ISSUE TITLE
Switching workspaces displays unwanted error message in the UI.

### ISSUE THREAD

**Issue by Pricilla09** [a85/POSTMan-Chrome-Extension#9748]
**Describe the Issue**
When we are trying to switch workspaces, the error "Could not find the request you are looking for" is displayed for few seconds and auto closed. But actually nothing is changed to the requests. Also the issue is sporadic.

**Steps to Reproduce**
When you have number of workspaces try to change from one workspace to another randomly.

**Screenshots or Videos**
![image](https://user-images.githubusercontent.com/71498483/111642063-b7e95380-8838-11eb-8b1c-a6fe6fbc3ee3.png)

**Environment Information**
 - Platform Type: Desktop application
 - Postman Version: Version 8.0.7
 - Operating System: Win 10

**Comment by timbochamp** [comment]
Hi @Pricilla09 , I noticed three of your tabs are blank, which could be that they were deleted requests. I think this might coincide with the three error messages you saw. I think @DannyDainton touched on this in the forums as well!

We made a change to close the tab of a deleted request to prevent this behaviour on a workspace switch, or closing and reopening the app. That said, I'm having trouble reproducing this on Windows 10 in the latest version (8.0.9) which addressed issues around these error messages. Are you able to update and report back if you're still seeing this issue?
